libs/sdk-backend-bear/src/backend/workspace/dashboards/index.ts

```diff
@@ -305,14 +306,26 @@ export class BearWorkspaceDashboards implements IWorkspaceDashboardsService {
         return toSdkModel.convertScheduledMail(createdBearScheduledMail) as IScheduledMail;
     };
 
+    public getScheduledMailsForDashboard = async (
+        dashboardRef: ObjRef,
+        createdByCurrentUser = false,
```

Comment:
We should likely hide `createdByCurrentUser` flag under some options object (so if we introduce more options in the future, eg for some other filtering, we can avoid parameters hell and breaking changes)

Reply:
Good point, fixed